parsec/core/sync_monitor.py

```diff
@@ -61,7 +61,7 @@ def running(self):
         with trio.CancelScope() as self._monitoring_cancel_scope:
             self.event_bus.send("sync_monitor.reconnection_sync.started")
             try:
-                await self.fs.full_sync()
+                await self.user_fs._syncer.full_sync()
```

Comment:
The `_syncer` object is legacy and shoudn't be used directly. What about `self.user_fs.sync()`?